pages/common/cancel.md

```diff
@@ -20,14 +20,14 @@
 
 `cancel {{printer}}-{{job_id}}`
 
-- Cancel all jobs of all printers:
+- Cancel [a]ll jobs of all printers:
 
 `cancel -a`
 
-- Cancel all jobs of a specific printer:
+- Cancel [a]ll jobs of a specific printer:
 
 `cancel -a {{printer}}`
 
-- Cancel the current job of a specific server and then delete job data files:
+- Cancel the current job of a specific server and then e[x]clude job data files:
```

Comment:
Delete and exclude both mean different things.

https://openprinting.github.io/cups/doc/man-cancel.html
> -x
> Deletes job data files in addition to canceling. 
```suggestion
- Cancel the current job of a specific server and then delete job data files:
```